From 1e7508ce083f6c7e43011f899faf10537a6379e2 Mon Sep 17 00:00:00 2001
From: Ivan Murashov <ivan.murashov@lge.com>
Date: Thu, 20 Feb 2025 10:37:19 -0800
Subject: [PATCH] IWYU: Add missing include for std::optional usage in
 picture_in_picture_window_manager_uma_helper.h

In the CL https://crrev.com/c/6191230 used std::optional in the
chrome/browser/picture_in_picture/picture_in_picture_window_manager_uma_helper.h,
but corresponding include was missed.
Added missing include for std::optional usage.

Bug: 41455655
Change-Id: I70f67c4a06481875d4602b16a97532cdad2a63b0
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6275068
Reviewed-by: Vasilii Sukhanov <vasilii@chromium.org>
Commit-Queue: Ivan Murashov <ivan.murashov@lge.com>
Reviewed-by: Tommy Steimel <steimel@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1422669}
---
 .../picture_in_picture_window_manager_uma_helper.h              | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/picture_in_picture/picture_in_picture_window_manager_uma_helper.h b/chrome/browser/picture_in_picture/picture_in_picture_window_manager_uma_helper.h
index cb826fc5a20598..866670224c8a38 100644
--- a/chrome/browser/picture_in_picture/picture_in_picture_window_manager_uma_helper.h
+++ b/chrome/browser/picture_in_picture/picture_in_picture_window_manager_uma_helper.h
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_PICTURE_IN_PICTURE_PICTURE_IN_PICTURE_WINDOW_MANAGER_UMA_HELPER_H_
 #define CHROME_BROWSER_PICTURE_IN_PICTURE_PICTURE_IN_PICTURE_WINDOW_MANAGER_UMA_HELPER_H_
 
+#include <optional>
+
 #include "base/memory/raw_ptr.h"
 #include "base/time/clock.h"
 #include "base/time/time.h"
